sktime/classification/deep_learning/esn.py

```diff
@@ -0,0 +1,159 @@
+__author__ = ["sharma-kshitij-ks"]
+
+from sklearn.base import BaseEstimator, ClassifierMixin

```

Comment:
please ensure you adhere to code formatting standards, the guide is here:
https://www.sktime.net/en/stable/developer_guide/coding_standards.html